src/include/miopen/convolution.hpp

```diff
@@ -74,6 +74,13 @@ using ExtraKernelArgs = std::tuple<int /*N*/,
 struct ConvFwdTensors;
 struct ConvWrwTensors;
 
+enum class ConvDirection
+{
+    Fwd = 1,
+    Bwd = 2,
+    WrW = 4
+};
+
```

Comment:
### It seems like the current design of unit tests requires moving entities from the local scope (implementation, e.g. *.cpp) to global scope (interface, e.g. *.hpp). This would lead to littering the code with unnecessary things and should be avoided.

For now, please revert this change. Put a copy of `ConvDirection` into log_test_helper.hpp instead -- just before declarations of `LogCmdConvolution()` and `LogCmdFindConvolution()`.